profile/README.md

```diff
@@ -0,0 +1,41 @@
+# OpenTimelineIO
+[![OpenTimelineIO](images/OpenTimelineIO@3xDark.png)](http://opentimeline.io)
```

Comment:
Adding a line that describes the open timeline io project in one sentence might be good here